pkg/storaged/stratis-details.jsx

```diff
@@ -263,6 +265,102 @@ export const StratisPoolDetails = ({ client, pool }) => {
         });
     }
 
+    function add_tang() {
+        const key_desc = pool.KeyDescription[1][1];
+        return client.stratis_list_keys()
+                .then(keys => {
+                    if (keys.indexOf(key_desc) >= 0)
+                        add_tang_with_keydesc(false);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230522-090630-61e0720a-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)